base/applications/rapps/appview.cpp

```diff
@@ -1892,6 +1892,39 @@ CApplicationView::AppendTabOrderWindow(int Direction, ATL::CSimpleArray<HWND> &T
     m_AppsInfo->AppendTabOrderWindow(Direction, TabOrderList);
 }
 
+VOID
+CApplicationView::GetRestoreListSelectionData(LVITEMW &Item, WCHAR *Name, UINT NameLen)
+{
+    Item.mask = LVIF_TEXT|LVIF_STATE;
+    Item.iItem = -1, Item.iSubItem = 0;
+    Item.stateMask = LVIS_FOCUSED|LVIS_SELECTED;
+    Item.pszText = Name, Item.cchTextMax = NameLen;
+
+    HWND hList = m_ListView ? m_ListView->m_hWnd : NULL;
+    if (hList)
+    {
+        Item.iItem = ListView_GetNextItem(hList, -1, LVNI_FOCUSED);
+        ListView_GetItem(hList, &Item);
```

Comment:
Why not use the convenience functions from the m_Listview itself?
stuff like `m_ListView->GetItemData` etc.

Reply:
I'm not a MFC guy I guess.

Reply:
You can find the reference here:
[https://doxygen.reactos.org/de/dfc/classCListView.html](https://doxygen.reactos.org/de/dfc/classCListView.html)